packages/components/src/avatar/src/AvatarGroup.tsx

```diff
@@ -70,6 +71,7 @@ export function InnerAvatarGroup({
     children,
     wrap = true,
     forwardedRef,
+    remainingAvatarsProps,
```

Comment:
Is this addition related to a specific issue?

Reply:
Related to the remainingAvatar issue while hovered.

Reply:
Could you elaborate please.

Reply:
The remainingAvatarsProps has been added to support adding a `data-testid` to the remainingAvatar dom element.

Reply:
Got it! Thanks